fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_coffin.dm

```diff
@@ -40,7 +40,7 @@
 
 /obj/structure/closet/crate/coffin/securecoffin
 	name = "secure coffin"
-	desc = "For insecure suckers to safely slumber in their secure sleeping space."
+	desc = "For insecure suckers to safely slumber in their secure sleeping space, surprisingly, this shelter's spacious sepulchre offers somewhat superior security"
```

Comment:
what is the joke supposed to be

Reply:
We dont make jokes, this is a serious community.